guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -68,39 +74,70 @@ If you lose connection to the command shell where the upgrade command is running
 ----
 # {foreman-maintain} upgrade run --target-version {TargetVersionMaintainUpgrade}
 ----
-
+endif::[]
+ifdef::katello[]
+. Check for running tasks
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake katello:upgrade_check
+----
+. Update operating system packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum update -y
+----
+. Update repositories
++
+.For Centos 7 or Red Hat Enterprise Linux Users:
+[options="nowrap" subs="attributes"]
+----
+# yum update -y https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+# yum install -y centos-release-scl-rh
+----
+. Update packages
+Clean the yum cache and update required packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum clean all
+# yum -y update
+----
+. Run installer
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-installer}
+----
+endif::[]
 . Check when the kernel packages were last updated:
 +
 [options="nowrap"]
 ----
 # rpm -qa --last | grep kernel
 ----
-
 . Optional: If a kernel update occurred since the last reboot, stop the `{foreman-maintain}` services and reboot the system:
```

Comment:
I don't understand this suggestion. Why do you need to stop all services and is just `reboot` insufficient? Is that always required? If so, IMHO it's a major limitation that we must think about.

Reply:
i agree, it doesn't make much sense.  Let me change to just 'reboot the system'.